dpe/src/commands/destroy_context.rs

```diff
@@ -57,8 +57,8 @@ impl CommandExecution for DestroyCtxCmd {
             .take(contexts_to_destroy.len())
         {
             if *context_to_destroy {
-                dpe.contexts[idx].destroy();
                 let parent = dpe.contexts[idx].parent_idx as usize;
+                dpe.contexts[idx].destroy();
```

Comment:
nit: in Context::destroy should we set parent_idx to 0xFF to protect against this code being moved around incorrectly in the future?

Reply:
Done